### ISSUE TITLE
Windows : after sync complete unable to read the fileSystem and load 

### ISSUE THREAD

**Issue by snehasis** [phonegap/phonegap-plugin-contentsync#128]
Hi,

I used the plugin and super in Android and IOS so far.
But in windows build after finishing unzip the folder to the path its unable to load that file in the App.

I am getting below error 
APPHOST9624: The app can’t use script to load the file:///C:UsersSsdsAppDataLocalPackages*_h35559jr9hy9mLocalStatestorewwwindex.html url because the url launches another app. Only direct user interaction can launch another app.

Basically I got the files unziped on the folder below.
C:\Users\USER\AppData\Local\Packages\PACKAGENAME\LocalState\store\

But while accessing this files Its not opening in the APP.

Please help.
Thanks

**Comment by dmaus** [comment]
I have the same problem. 
`APPHOST9624: The app can’t use script to load the  "ms-appdata:///local/myapp/index.html" url because the url launches another app. Only direct user interaction can launch another app.`
I tried to set `<allow-navigation href="*" />` in config.xml but it doesn't work. Any ideas?

**Comment by macdonst** [comment]
I think you are running into a limitation of the Windows platform. Let me see if I can get someone to answer this definitively for you.

**Comment by dmaus** [comment]
OK thanks. That would be bad, because the whole "update over the air" solution wont work for windows...

**Comment by purplecabbage** [comment]
Unfortunately Windows does not allow this functionality.
